tasks/30-cache.yml

```diff
@@ -5,8 +5,15 @@
   tags:
     - cache
 
-- name: Warmup sf cache.
+- name: Warmup sf cache without environment variables
```

Comment:
staying with the current code, you should add a "." at the end. Just a minor thing - but form is important :-)